### PR TITLE
Update UG with AddCoursesCommand 

### LINKED ISSUE #193: Update User Guide for AddCoursesCommand

### PR DESCRIPTION
**Pull request by hzxnancy**
Fixes #193 

Add description and implementation of AddCoursesCommand in the User Guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-Introducing ExchangeCourseMapper, the perfect assistant in your planning for your SEP in Australia!
+Introducing ExchangeCourseMapper, the perfect assistant in your planning for your SEP in Oceania!
```
Comment (chewycj) [inline docs/UserGuide.md:5]: Thanks for catching this! 🥇

**Review by chewycj — APPROVED** [review]
LGTM! Clear documentation especially for the more complicated add command! 🥇

File: docs/UserGuide.md [hunk 2]
```diff
@@ -20,21 +20,49 @@ you can save it in the Personal Tracker provided by ExchangeCourseMapper!
...
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+Format: `add NUS_COURSE_CODE /pu PARTNER_UNIVERSITY_NAME /coursepu PU_COURSE_CODE`
 
-Format: `todo n/TODO_NAME d/DEADLINE`
+* All 3 parameters `NUS_COURSE_CODE`, `PARTNER_UNIVERISTY_NAME` and `PU_COURSE_CODE` are case-insensitive.
+* Do not add punctuation to the above three parameters
+* Indicate the full name of the partner university for `PARTNER_UNIVERISTY_NAME`. For example, indicate
+`The Australian National University` instead of `Australian National University` or `ANU`.
```
Comment (louisjoety) [inline docs/UserGuide.md:33]: May need to review some merge conflicts

Comment (hzxnancy) [reply docs/UserGuide.md:79]: Thanks for pointing it out! I have resolved the conflicts with the new commit

**Pull request merged**